pkgs/top-level/all-packages.nix

```diff
@@ -23280,8 +23280,6 @@ with pkgs;
     tpmSupport = true;
   };
 
-  ovmfvartool = callPackage ../applications/virtualization/ovmfvartool { };
```

Comment:
We probably want to keep this, to not break some people using it as an application

```
  ovmfvartool = with python3.pkgs; toPythonApplication ovmfvartool;
```

Reply:
it was included only a few days ago. I don't think this is too bad. I think I'm only consumer for now :D